RFC-0030-native-fp8-dtype.md

```diff
@@ -0,0 +1,222 @@
+# Proposal of fp8 dtype introduction to PyTorch
+
+**Authors:**
+* @australopitek
+
+
+## **Summary**
+More and more companies working on Deep Learning accelerators are experimenting with 8-bit floating point numbers usage in training and inference. Results of these experiments are presented in many papers published in the last few years.
+
+Since fp8 data type seems to be a natural evolution of currently used fp16/bf16, to reduce computation of big DL models, it’s worth to standardize this type. Few attempts of this were done recently:
+
+* Nvidia, Arm and Intel - https://arxiv.org/pdf/2209.05433.pdf
+* GraphCore and AMD - https://arxiv.org/pdf/2206.02915.pdf
```

Comment:
For completeness, these formats are proposed by Graphcore, AMD, and Qualcomm.

Reply:
I'll correct it when more comments are there.